File: Unidad 2-Estructuras de Control/Ejercicio1.c
/*
Ejercicio 1 C
Unidad 2 - {Programación Estructurada}
Equipo 1
Autor: Daniel Eduardo Gutierrez Delfin
Fecha: 8 de Febrero 2019
Entradas: Un número
Salidas: Si el numero es par o impar
---------------------------------------
Instrucción:
Escribe un programa que reciba un número y te diga si ese número es par o
impar.
---------------------------------------
Procedimiento General:
---------------------------------------
1.Se define una variable
2.Se le proyecta al usuario que debe ingresar un numero
3.Se lee el numero y se convierte en la variable
4.Divimos el numero entre 2 y vemos si su residuo es igual a cero.
5 Si es igual a cero entonces se imprime que el numero es par.
6.Si es diferente de cero entonces se imprime que el numero es impar.
---------------------------------------
*/
#include <stdio.h>

int main()
{
	//- *Entradas*//
	//Definimos n como variable// 
  int numero; 
  int esPar = 0;

	printf ("Escribe un numero entero \n"); //Se da el mensaje al usuario//
	scanf ("%d", &numero); //leemos numero//

	//- *Procedimineto//
  //Vemos si el residuo de n dividido entre 2 es igual a 0 - *Procedimineto//
	if (numero % 2 == 0){
		esPar = 1;
  } else {
		esPar = 0;
  }
	//- *Salidas*//
//Se imprime el resultado 
  if(esPar){
    printf("El numero %d es par", numero);
  } else {
    printf("El numero %d es impar", numero);
  }
	return 0;
}

/*
Autor QA:Emmanuel Azcorra Balam
Entradas:3,4,3,12
salidas:impar,par,impar,par
Proceso :Ok, todo correcto.*/

File: Unidad 2-Estructuras de Control/Ejercicio1.py
#Ejercicio 1 
#Escribe un programa que reciba un número y te diga si ese número es par o impar
#Autor: Pamela Citlali Canul Chacón (Equipo 'about:blank' )

#Dato de entrada: Un número
#Dato de salida: Si el número es par o impar

#Entrada
#Se definen las variables, una donde se guardara el valor del número a analizar y la otra servira como bandera
num = 0 
esPar = 0
#Se le solicita al usuario ingresar un número
num = int(input("Ingresa un número: "))

#Procedimiento
#Se evalua el residuo por medio de una condicional
if num % 2 == 0 :
    #Si el resultado es verdadero se cambia el valor de la bandera
    esPar = 1 
else:
    esPar = 0

#Salida
#Se imprime el resultado dependiendo del valor de la bandera
if esPar == 1: 
    print ("El número ", num, "es par")
else: 
    print ("El número ", num, "es impar") 
